prow/plugins/verify-owners/verify-owners.go

```diff
@@ -222,10 +222,6 @@ func handle(ghc githubClient, gc git.ClientFactory, roc repoownersClient, log *l
 		}
 	}
 
-	if len(modifiedOwnersFiles) == 0 && !ownerAliasesModified {
```

Comment:
if you remove this condition, then you will clone the repo for every PR...

Reply:
it would be better to add a condition to still verify owners whenever `ownersFileName` or `ownersAliasesFileName` have a `change.Status` of `github.PullRequestFileRemoved`

Reply:
Hi @matthyx. Thanks for your quick response!
Good point about repo cloning, Returned back that short-circuiting with an additional check if there were any owner files modifications.